lib/embroider/update-manifest-from-html.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const fs = require('fs');
+const findScriptSrcs = require('find-scripts-srcs-in-document');
+const isEmbroider = require('./is-embroider');
+
+// TODO: hard coding these seem unfortunate, especially vendor-static which is non-standard
```

Comment:
is there an alternative, hardcoding these seems brittle/error prone.

Reply:
I don't know if there are any alternatives to this approach other than using the attribute on script tags indicating skip for fastboot.

Reply:
> other than using the attribute on script tags indicating skip for fastboot.

Yup this sounds appropriate.

Reply:
Instead of hardcoding the vendor files to skip, I have added a check to verify if the file from index.html is present in `manifest.vendorFiles` and skip them. 

This will ensure we are only checking for App files and not including any vendor files.